custom/icds/location_reassignment/views.py

```diff
@@ -63,8 +64,17 @@ def post(self, request, *args, **kwargs):
         return self.get(request, *args, **kwargs)
 
     def _workbook_is_valid(self, workbook):
-        # ToDo: Add necessary checks for workbook
-        return []
+        # ensure worksheets present and with titles as the location type codes
```

Comment:
Should this also check that there are no missing tabs?

Reply:
I had that earlier but i am currently keeping things flexible because i might be asked to not need all sheets and just relevant ones/ones that change.